video/out/vo_gpu_next.c

```diff
@@ -2321,9 +2321,6 @@ AV_NOWARN_DEPRECATED(
         break;
     }
 
-    if (opts->dither_depth < 0)
-        pars->params.dither_params = NULL;
```

Comment:
is this assured or checked by something else?

Reply:
We set the dither depth parameters here:
https://github.com/mpv-player/mpv/blob/d7668755876d956ba3a8295534800b7ba0cede60/video/out/vo_gpu_next.c#L829-L842

It's more involved logic, so it should be in one place.

Reply:
ok but it looks like that won't reset anything if `dither_depth < 0`

Reply:
Fair. Indeed it is excluded here. Added it back and fixed for error diffusion.